src/main/resources/jsonSchema/clientSettings.schema.json

```diff
@@ -192,6 +192,19 @@
         }
       }
     },
+    "references": {
```

Comment:
And finally, updated the client config JSON schema for this feature flag.

Note that I have not automatically enabled this feature _anywhere_ including the TypeScript language server definition template. Unless explicitly enabled somewhere, this feature is inactive. Since I know that my own plugin can contribute external references to JavaScript, TypeScript, and CSS, I will enable it in the integration managed by my plugin. That will limit the potential risk around restoring this while making it available to integrations where it's known to be valuable.